path_2023.py

```diff
@@ -57,7 +59,20 @@ def run() -> int:
     if not result:
         print("Error: upload_commands")
         return -1
-    
+
+    waypoint_info = waypoint_tracking.get_current_waypoint_info(drone)
+    if not waypoint_info:
+        print("Error: waypoint_tracking")
+        return -1
+
+    while True:
+        current_latitude, current_longitude = waypoint_tracking.get_current_location(drone)
+        if not current_latitude:
+            print("Error: waypoint_tracking")
+            return -1
+
+        time.sleep(0.1)
```

Comment:
Move value into a global constant at the top of the file (e.g. `DELAY = 0.1  # seconds` ).